git-essentials.md

```diff
@@ -0,0 +1,34 @@
+# Introduction to Git
+
+1. git config
+
+   - The git config command is used to configure various aspects of Git, including user information, repository settings, and more.
+     For Example:git config --list,git config user.name etc
+
+2. git clone "https:....."
+
+   - Used for cloning repo locally
+
+3. git status
+
+   - to know the branch is up to date or not
+
+4. git add
+
+   - to stage the changes for next commit
+
+5. git commit
+
+   - To commit the changes in the branch
+
+6. git push
+   - to commit local changes into remote repo
+
+Some other commands to work in terminal
```

Comment:
These should go in a separate file 
you can name it terminal.md
title can be working with unix terminal